src/CloudNimble.DotNetDocs.Sdk/CloudNimble.DotNetDocs.Sdk.csproj

```diff
@@ -19,65 +21,55 @@
         
         <!-- Suppress dependencies when packing -->
         <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddPerTFMDocsTasksToPackage</TargetsForTfmSpecificContentInPackage>
     </PropertyGroup>
 
     <ItemGroup>
-        <!-- Include Microsoft.Build.NoTargets for bundling its SDK files -->
-        <PackageReference Include="Microsoft.Build.NoTargets"
-                          Version="3.7.*"
-                          PrivateAssets="all"
-                          GeneratePathProperty="true" />
+        <!-- MSBuild task dependencies -->
+        <PackageReference Include="EasyAF.MSBuild" Version="3.*-*" />
+        <PackageReference Include="Microsoft.Build.Framework" Version="17.*" PrivateAssets="all" />
+        <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.*" PrivateAssets="all" ExcludeAssets="runtime" />
+        <PackageReference Include="System.Collections.Immutable" Version="9.*" PrivateAssets="all" />
+        
+        <!-- Microsoft.CodeAnalysis.CSharp.Workspaces needs to be available for the task -->
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.*" />
```

Comment:
nit: without a lock file, all of these dependencies are nondeterministic. This is _bad_ for consumers of your MSBuild SDK. You should either use a lockfile with these or move to single-version references here so that consumers of your SDK know what they're getting.